src/element-templates/ElementTemplates.js

```diff
@@ -196,24 +196,35 @@ export default class ElementTemplates {
    * @return {boolean} - true if compatible or no engine is set for elementTemplates or template.
    */
   isCompatible(template) {
+    return !Object.keys(this.getIncompatibleEngines(template)).length;
+  }
+
+  /**
+   * Get engines that are incompatible with the template.
+   *
+   * @param {ElementTemplate} template
+   *
+   * @return {Object} - incompatible engines along with their template and local versions
+   */
+  getIncompatibleEngines(template) {
```

Comment:
If we expose another public API then we need to test this.

Reply:
Added with https://github.com/bpmn-io/bpmn-js-element-templates/pull/132/commits/54da72d8601a4af889f53738405aaee370165e9e